As investigated in https://github.com/kuthulux/gnome-connection-manager/issues/5
and https://github.com/kuthulux/gnome-connection-manager/pull/7 the default
TERM value of xterm-256color did not work well for some applications rendered
in GCM VTEs in OI. While the upstream may develop a better fix for this area,
a hot fix to have this out of the box is to default a TERM which works better.

--- github/gnome_connection_manager.py	2020-04-15 11:56:25.772356289 +0000
+++ fixed/gnome_connection_manager.py	2020-04-15 12:39:05.024238615 +0000
@@ -450,7 +450,7 @@
         terminal.feed_child(data, len(data))
 
 def vte_run(terminal, command, arg=None):
-    envv = [ 'PATH=%s' % (os.getenv("PATH")) ]
+    envv = [ 'PATH=%s' % (os.getenv("PATH")), 'TERM=%s' % (os.getenv("TERM")) ]
     args = []
     args.append(command)
     if arg:
@@ -1145,7 +1145,10 @@
             v.set_scrollback_lines(conf.BUFFER_LINES)
             #if v.get_emulation() != os.getenv("TERM"):
             #    os.environ['TERM'] = v.get_emulation()
+            ### As of Apr 2020 there are issues in OI use of xterm* in gcm
+            ### https://github.com/kuthulux/gnome-connection-manager/issues/5
-            os.environ['TERM'] = 'xterm-256color'
+            #os.environ['TERM'] = 'xterm-256color'
+            os.environ['TERM'] = 'screen-256color'
             
             if isinstance(host, str):
                 host = Host('', host) 
